crates/pathfinder/src/config/builder.rs

```diff
@@ -41,6 +41,14 @@ impl ConfigBuilder {
         // Required parameters.
         let eth_url = self.take_required(ConfigOption::EthereumHttpUrl)?;
 
+        // this used to be the url in docker run example
+        if eth_url == "https://goerli.infura.io/v3/<project-id>" {
+            return Err(std::io::Error::new(
+                std::io::ErrorKind::InvalidInput,
+                format!("Invalid Ethereum URL ({eth_url}): Cannot use the URL from examples! Register your own account or run your own Ethereum node and put the real URL as the configuration value.")
```

Comment:
```suggestion
                format!("Invalid Ethereum URL ({eth_url}): Cannot use the URL from examples!

Hint: Register your own account or run your own Ethereum node and put the real URL as the configuration value.")
```

This would give

```
Error: Parsing configuration

Caused by:
    Invalid Ethereum URL (https://goerli.infura.io/v3/<project-id>): Cannot use the URL from examples!

    Hint: Register your own account or run your own Ethereum node and put the real URL as the configuration value.
```

I think this is good enough? Makes it more readable and perhaps more obvious.

Reply:
Yeah I think this is good.